access/src/main/external/static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -134,8 +134,13 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'E
 			}
 		}
 		
+		if ($.inArray('changeResourceType', metadata.permissions) != -1
+				&& $.inArray('info:fedora/cdr-model:Container', metadata.model) != -1) {
+			items["changeType"] = {name : 'Change Type'};
```

Comment:
Also, could this be called "Edit Type" instead? Why is this "change" but the other options are "edit"?